docs/tiltfile_concepts.md

```diff
@@ -222,9 +222,11 @@ resource, note that `new_name` is required.)
 
 #### Kubernetes Object Selectors
 
-You can specify Kubernetes objects via a _Kubernetes object selector_. (NOTE: 
-this is a Tilt-specific syntax. We wish Kubernetes already had a standard for
-specifying objects, but they don't, so we made our own.)
+When specifying the `objects` param to `k8s_resource`, you can specify Kubernetes objects via a _Kubernetes object selector_.
+
+Notes:
+1. This is a Tilt-specific syntax. We wish Kubernetes already had a standard for specifying objects, but they don't, so we made our own.
```

Comment:
i don't think this is accurate?
https://kubernetes.io/docs/concepts/overview/working-with-objects/field-selectors/
https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/

i think the author of this disagreed with kuberentes' opinions about selectors and how they work, and so invented a new concept of identifier

Reply:
I don't know if that's really an argument I want to come up to speed on and weigh in on while fixing this different problem, but I've just deleted the "We wish.." sentence.